besides that it's ugly, it also breaks the build since the Apple
toolchain is broken enough to hit some assertion when compiling with
debug enabled

--- base/macos-fw.mak
+++ base/macos-fw.mak
@@ -68,7 +68,7 @@
 # it would be nice if we could link to the framework instead
 
 $(GSSOC_XE): $(GS_SO) $(GLSRC)dxmainc.c
-	$(GLCC) -g -o $(GSSOC_XE) $(GLSRC)dxmainc.c -L$(BINDIR) -l$(GS)
+	$(GLCC) -o $(GSSOC_XE) $(GLSRC)dxmainc.c -L$(BINDIR) -l$(GS)
 
 # ------------------------- Recursive make targets ------------------------- #
 
--- base/unix-dll.mak
+++ base/unix-dll.mak
@@ -81,10 +81,10 @@
 # Build the small Ghostscript loaders, with Gtk+ and without
 
 $(GSSOC_XE): $(GS_SO) $(PSSRC)$(SOC_LOADER)
-	$(GLCC) -g $(LDFLAGS) -o $(GSSOC_XE) $(PSSRC)dxmainc.c -L$(BINDIR) -l$(GS)
+	$(GLCC) $(LDFLAGS) -o $(GSSOC_XE) $(PSSRC)dxmainc.c -L$(BINDIR) -l$(GS)
 
 $(GSSOX_XE): $(GS_SO) $(PSSRC)$(SOC_LOADER)
-	$(GLCC) -g $(LDFLAGS) $(SOC_CFLAGS) -o $(GSSOX_XE) $(PSSRC)$(SOC_LOADER) \
+	$(GLCC) $(LDFLAGS) $(SOC_CFLAGS) -o $(GSSOX_XE) $(PSSRC)$(SOC_LOADER) \
 	-L$(BINDIR) -l$(GS) $(SOC_LIBS)
 
 # ------------------------- Recursive make targets ------------------------- #
